src/chorizo-sweet-potato-enchiladas.md

```diff
@@ -0,0 +1,37 @@
+# Chorizo and Sweet Potato Enchiladas
+
+Very filling with a sweet and spicy flavour.
+
+## Ingredients
+
+- 1 large sweet potato (approx 1 pound)
+- 1 bell pepper
+- 1 jalapenos (adjust based on spice preference)
+- 2 cloves garlic
+- vegetable oil
```

Comment:
Quantify vegetable oil?

Reply:
The reason I didn't quantify it is that personally I never put a measured amount of oil in the pan as for me it's mostly there to stop stuff sticking to the pan, and so the amount varies based on the quality of non-stick.

(The oil is quantified in the sauce recipe as it affects the consistency of the sauce.)